### PR TITLE
Add balances to activity screens

### LINKED ISSUE #875: Update activity screens to include the wallet balance (daily spending wallet)
We focused the [home screen](http://localhost:4000/guide/daily-spending-wallet/sending/#payment-entry-points) on initiating a transaction and removed the balance for privacy purposes. But we didn't add the balance anywhere else (!?). I can go over all the screens and add it to the [activity](http://localhost:4000/guide/daily-spending-wallet/activity/) screen (seems logical to have it there).

Example:
activity-labeling

### PR DESCRIPTION
**Pull request by GBKS**
🧌[Preview](https://deploy-preview-876--bitcoin-design-site.netlify.app/guide/daily-spending-wallet/activity/)👣

Closes #875

### CODE REVIEW

File: _compress_images_cache.yml [hunk 1]
```diff
@@ -1524,122 +1524,6 @@ assets/images/projects/projects@2x.jpg: projects@2x.jpg
...
-assets/animations/astronaut/images/img_0.png: img_0.png
-assets/animations/astronaut/images/img_1.png: img_1.png
-assets/animations/contribute/formatting/bitcoin-design-seal-miami-edition.png: bitcoin-design-seal-miami-edition.png
-assets/animations/contribute/formatting/bitcoin-design-seal.png: bitcoin-design-seal.png
-assets/animations/contribute/formatting/bitcoin-design-seal-miami-edition@2x.png: bitcoin-design-seal-miami-edition@2x.png
-assets/animations/contribute/formatting/bitcoin-design-seal@2x.png: bitcoin-design-seal@2x.png
-assets/images/home/banner/bits-and-pieces-fallback.png: bits-and-pieces-fallback.png
-assets/images/home/banner/bits-and-pieces-fallback@2x.png: bits-and-pieces-fallback@2x.png
-assets/images/guide/resources/design-files/bitcoin-wallet-backup-print-template.jpg: bitcoin-wallet-backup-print-template.jpg
-assets/images/guide/resources/design-files/bitcoin-wallet-backup-print-template@2x.jpg: bitcoin-wallet-backup-print-template@2x.jpg
```
Comment (sbddesign) [inline _compress_images_cache.yml:1642]: Why are these being removed from the compress cache?

**Review by sbddesign — CHANGES_REQUESTED** [review]
The update to the screens looks good. I noticed a lot of lines have been removed from the compress images cache. Is this an error?

File: _compress_images_cache.yml (hunk 1 above)
Comment (GBKS) [reply _compress_images_cache.yml:1642]: See [my other comment](https://github.com/BitcoinDesign/Guide/pull/873#discussion_r929545341).

**Review by Bosch-0 — DISMISSED** [review]
Looks good! I think there is lots of stuff we need to update in the UI kit, once this milestone is wrapped up lets all do a thorough review of it

**Review by Bosch-0 — APPROVED** [review]

**Comment by mouxdesign** [comment]
Love the design iterations for these!

**Review by sbddesign — APPROVED** [review]
🌵

**Pull request merged**